CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Release Date: _unreleased_
 - Reworked internal mechanism used to fetch network configuration. Is no longer stored in the underlying database and it is shared across all services.
 
 ### What’s new?
+- Added `manifest_classes` collection to `CommittedTransactionInfo`: a collection of zero or more manifest classes ordered from the most specific class to the least specific one.
```

Comment:
can we add information which endpoints are affected?

Reply:
updated the changelog